From: Marco Nenciarini <mnencia@debian.org>
Date: Fri, 16 Sep 2011 00:55:02 +0200
Subject: fix mail_plugin_dir default

Bug: #624294

Fix mail_plugin_dir default value in conf.d/10-mail.conf
---
 doc/example-config/conf.d/10-mail.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/example-config/conf.d/10-mail.conf b/doc/example-config/conf.d/10-mail.conf
index de48f92..24daa04 100644
--- a/doc/example-config/conf.d/10-mail.conf
+++ b/doc/example-config/conf.d/10-mail.conf
@@ -211,7 +211,7 @@ namespace inbox {
 #auth_socket_path = /var/run/dovecot/auth-userdb
 
 # Directory where to look up mail plugins.
-#mail_plugin_dir = /usr/lib/dovecot
+#mail_plugin_dir = /usr/lib/dovecot/modules
 
 # Space separated list of plugins to load for all services. Plugins specific to
 # IMAP, LDA, etc. are added to this list in their own .conf files.
